packit_service/service/events.py

```diff
@@ -61,6 +62,12 @@ class PullRequestAction(enum.Enum):
     synchronize = "synchronize"
 
 
+class MergeRequestAction(enum.Enum):
+    opened = "opened"
+    reopen = "reopen"
+    update = "update"
```

Comment:
I am not against but I am thinking whether it is possible to reuse some existing enum for that.

e.g. this one:

https://github.com/packit-service/ogr/blob/fda65c62ee039786f223433367a8c64d353f4692/ogr/abstract.py#L276-L280

(But I know that you would still need to remap those two.